Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -88,9 +88,12 @@ struct HFTrackIndexSkimsCreator {
 
   OutputObj<TH1F> hvtx_x_out{TH1F("hvtx_x", "2-track vtx", 1000, -2.0, 2.0)};
   OutputObj<TH1F> hvtx_y_out{TH1F("hvtx_y", "2-track vtx", 1000, -2.0, 2.0)};
-  OutputObj<TH1F> hvtx_z_out{TH1F("hvtx_z", "2-track vtx", 1000, -10.0, 10.0)};
+  OutputObj<TH1F> hvtx_z_out{TH1F("hvtx_z", "2-track vtx", 1000, -20.0, 20.0)};
   OutputObj<TH1F> hmass2{TH1F("hmass2", "; Inv Mass (GeV/c^{2})", 500, 0, 5.0)};
   OutputObj<TH1F> hmass3{TH1F("hmass3", "; Inv Mass (GeV/c^{2})", 500, 1.6, 2.1)};
+  OutputObj<TH1F> hvtx3_x_out{TH1F("hvtx3_x", "3-track vtx", 100, -1.0, 1.0)};
+  OutputObj<TH1F> hvtx3_y_out{TH1F("hvtx3_y", "3-track vtx", 100, -1.0, 1.0)};
+  OutputObj<TH1F> hvtx3_z_out{TH1F("hvtx3_z", "3-track vtx", 100, -20.0, 20.0)};
```

Comment:
@DelloStritto. As mentioned in the validation's PR, we could make the ranges consistent among 2 and 3 prongs. 
Then we will change the AliPhysics PR accordingly.